core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -671,6 +782,57 @@ class KafkaApisTest {
     }
   }
 
+  @Test
+  def shouldReplaceTxnTimeoutWithInvalidProducerEpochInEndTxnWithOlderClient(): Unit = {
```

Comment:
Could we consolidate the tests with `shouldReplaceProducerFencedWithInvalidProducerEpochInEndTxnWithOlderClient`? Similar to other error code replacement tests.

Reply:
Correct me if I misunderstood your comment.  
Error code replacement tests with TxnTimeout have beed added which is followed.
shouldReplaceTxnTimeoutWithInvalidProducerEpochInAddOffsetToTxnWithOlderClient
shouldReplaceTxnTimeoutWithInvalidProducerEpochInAddPartitionToTxnWithOlderClient
shouldReplaceTxnTimeoutWithInvalidProducerEpochInEndTxnWithOlderClient